src/components/Jam.jsx

```diff
@@ -31,6 +39,61 @@ const ValuesListener = ({ handler }) => {
   return null
 }
 
+const DEFAULT_PRECONDITION_SUMMARY = {
+  isFulfilled: false,
+  numberOfMissingUtxos: SCHEDULER_PRECONDITIONS.MIN_NUMBER_OF_UTXOS,
+  amountOfMissingConfirmations: SCHEDULER_PRECONDITIONS.MIN_CONFIRMATIONS_OF_SINGLE_UTXO,
+  amountOfMissingOverallRetries: SCHEDULER_PRECONDITIONS.MIN_OVERALL_REMAINING_RETRIES,
+}
+
+const useSchedulerPreconditionSummary = (walletInfoOrNull, startAccountIndex) => {
+  const eligibleUtxos = useMemo(() => {
+    if (!walletInfoOrNull) return []
+
+    const utxos = walletInfoOrNull.data.utxos.utxos || []
+    return utxos
+      .filter((it) => it.mixdepth === startAccountIndex)
+      .filter((it) => !it.frozen)
+      .filter((it) => !it.locktime)
```

Comment:
Can frozen UTXOs _not_ be the source of a comitment?

Reply:
> Can frozen UTXOs _not_ be the source of a comitment?

I am not super sure, but I do not think so. Reading [Adam's blog post about P(o)ODLE](https://reyify.com/blog/poodle), it says:
> Later on in the protocol you will open the commitment and reveal the utxo.

Frozen utxos wont take part in the collaborative transaction and can therefore not be used in the commitment. Does that make sense?

Reply:
> Frozen utxos wont take part in the collaborative transaction and can therefore not be used in the commitment.

Not sure if that's the implication from that. Theoretically all you want to do is prove that you have some UTXO right? But I guess if we apply stricter restrictions and they turn out to be too strict we can always loosen them later.